styleguide/package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "start": "start-storybook -p 9001",
     "build-storybook": "build-storybook",
-    "build": "rm -rf dist && tsc",
+    "build": "rimraf -rf dist && tsc",
```

Comment:
Do we need to add `rimraf` to this package's dependencies too?

Reply:
good catch, this needs to be added